Empire/Empire.cpp

```diff
@@ -1399,6 +1411,30 @@ void Empire::AddPartType(const std::string& name) {
         return;
     m_available_part_types.insert(name);
     AddSitRepEntry(CreateShipPartUnlockedSitRep(name));
+
+    // Decide whether to show the tutorial sitrep
+    // about building outpost or colony ships.
+    if (name == "CO_OUTPOST_POD") {
+        DebugLogger() << "Empire::AddPartType CO_OUTPOST_POD";
+        if (m_available_part_types.find("CO_COLONY_POD") != m_available_part_types.end())
+            return;
+    } else if (name == "CO_COLONY_POD") {
+        DebugLogger() << "Empire::AddPartType CO_COLONY_POD";
+        if (m_available_part_types.find("CO_OUTPOST_POD") != m_available_part_types.end())
+            return;
+    } else
+            return;
+    for (auto hull_name = m_available_hull_types.cbegin();
+            hull_name != m_available_hull_types.cend(); hull_name++) {
```

Comment:
odd indenting

prefer `{` on its own line after multi-line for

Reply:
This code will likely change when switching to FOCS scripting, so i'm postponing the style change until that is resolved.

Reply:
This code was deleted in commit f4d47c23, so i think this conversation can be resolved.